### ISSUE TITLE
Tabs are not visible on next or preview action

### ISSUE THREAD

**Issue by mladenbabic** [stepstone-tech/android-material-stepper#12]
Hi, tnx for this great lib.
I have a problem with a tab visibility.
f.e. If you have lots of tabs and some of them are off screen, the next or prev button does not slide (show) to the current page (tab) and you have to manually scroll to it.
 
Kind regards

**Comment by mladenbabic** [comment]
I have found a quick fix until you officially fix the bug.

Just replace the setCurrentStep() method in the TabsContainer class with this:

```
public void setCurrentStep(int newStepPosition) {
        int size = mStepTitles.size();
        for (int i = 0; i < size; i++) {
            StepTab childTab = (StepTab) mTabsInnerContainer.getChildAt(i);
            boolean done = i < newStepPosition;
            final boolean current = i == newStepPosition;
            childTab.updateState(done, current);
           
            HorizontalScrollView hv = (HorizontalScrollView) mTabsInnerContainer.getParent();
            if(current) {
                hv.smoothScrollTo(childTab.getLeft() - 20, 0);
            }
        }
    }
```
It works ok for me.

**Comment by zawadz88** [comment]
Thx for the feedback, we'll investigate :)

**Comment by zawadz88** [comment]
Fixed and published in v1.0.2

**Comment by mladenbabic** [comment]
Tnx a lot :)
